### ISSUE TITLE
执行脚本报错minitouch setup timeout

### ISSUE THREAD

**Issue by wjhh** [AirtestProject/Airtest#139]
执行脚本时有如下报错，为了确认不是自己脚本问题，我还单独录制了一个点击事件并执行，也是同样的报错。
客户端：AirtestIDE_2018-08-07_py3_win64（v0.1.6）
设备：htc m8，已root
log：
```
ERROR: runTest (app.widgets.code_runner.ide_launcher.AirtestIDECase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "airtest\cli\runner.py", line 66, in runTest
  File "site-packages\six.py", line 693, in reraise
  File "airtest\cli\runner.py", line 62, in runTest
  File "E:\AirtestIDE_2018-08-07_py3_win64\test.air\test.py", line 39, in <module>
    poco("<Layer | Tag = -1>").child("<Node | Tag = -1").child("<Sprite | Tag = -1, TextureID = 89>")[0].click()
  File "poco\proxy.py", line 23, in wrapped
  File "poco\proxy.py", line 285, in click
  File "poco\pocofw.py", line 248, in click
  File "poco\utils\airtest\input.py", line 62, in click
  File "airtest\utils\logwraper.py", line 72, in wrapper
  File "airtest\core\api.py", line 261, in touch
  File "airtest\core\android\android.py", line 330, in touch
  File "airtest\utils\snippet.py", line 104, in ready_func
  File "airtest\utils\snippet.py", line 115, in wrapper
  File "airtest\core\android\minitouch.py", line 51, in install_and_setup
  File "airtest\core\android\minitouch.py", line 143, in setup_server
RuntimeError: minitouch setup timeout
```

**Comment by yimelia** [comment]
@wjhh 可以查看 [faq](http://airtest.netease.com/docs/docs_AirtestIDE-zh_CN/8_faq/2_android_faq.html#airtest) 获得更多帮助：

如果是已root的手机，可能会因为本地文件夹权限已被修改，导致Airtest没有权限将相关文件放入手机内，请检查手机内的 /data/local/tmp 文件夹是否有写入权限
如果没有写入权限，请chmod该文件夹的权限，添加w权限让airtest能顺利安装完成

**Comment by wjhh** [comment]
drwxrwxrwx shell    shell             2018-08-21 10:16 tmp
我这个文件夹应该是有写入权限吧，adb也可以push文件到文件夹里面
看输出log应该是写入minicap成功了？但是为什么skip install minicap了，和这个有关系吗
[10:27:42][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell ls /data/local/tmp/minicap ; echo ---$?---
[10:27:42][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell ls /data/local/tmp/minicap.so ; echo ---$?---
[10:27:42][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell LD_LIBRARY_PATH=/data/local/tmp /data/local/tmp/minicap -v 2>&1
[10:27:42][DEBUG]<airtest.core.android.minicap> version:5
[10:27:42][DEBUG]<airtest.core.android.minicap> skip install minicap
[10:27:42][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell LD_LIBRARY_PATH=/data/local/tmp /data/local/tmp/minicap -i ; echo ---$?---
[10:27:42][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell pm path jp.co.cyberagent.stf.rotationwatcher ; echo ---$?---
[10:27:43][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 shell export CLASSPATH=/data/app/jp.co.cyberagent.stf.rotationwatcher-1/base.apk;exec app_process /system/bin jp.co.cyberagent.stf.rotationwatcher.RotationWatcher
[10:27:43][DEBUG]<airtest.core.android.adb> E:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows\adb.exe -P 5037 -s HT49JWM02255 forward --no-rebind tcp:13691 localabstract:minicap_13691

**Comment by yimelia** [comment]
skip的是minicap，安装过的话就会skip。
在IDE上面，连了手机以后，能在IDE里看到手机画面并且用鼠标来操作屏幕吗？

**Comment by SUDARSHANCHAUDHARI** [comment]
Try to copy - paste the files from"C:\Old Version of Airtest IDE\airtest\core\android\static\adb\windows" to "C:\AirtestIDE_2018-08-07_py3_win64\airtest\core\android\static\adb\windows"

**Comment by SUDARSHANCHAUDHARI** [comment]
I setup Airtest on two machines (One is much more advanced compared to second).
New machine and Old machine from onwards.

In the beginning, I used airtest Version (V0.0.18) on both machines,
and then at the time of new version release, I switch to new version Airtest (V0.1.6) (AirtestIDE_2018-08-07_py3_win64).

That version works on the new machine but shows that ADB related issue on the old machine.
I don't remember the exact error but something file missing from a particular folder (\airtest\core\android\static\adb\windows).

I checked both versions and both folders /files.
to avoid any confusion I just copy all files from an older version and paste it in the newer version of airtest.
Surprisingly issue solved.
